src/utils/DepositUtils.ts

```diff
@@ -71,3 +73,38 @@ export async function queryHistoricalDepositForFill(
 
   return validateFillForDeposit(fill, deposit) ? deposit : undefined;
 }
+
+export function createDepositForSimulatingGas(
+  amountToRelay: BigNumberish,
+  tokenSymbol: string,
+  originChainId: string,
+  destinationChainId: string,
+  relayerAddress: string,
+  messagePayload?: {
+    message: string;
+    recipientAddress: string;
+  }
+): Deposit {
+  const originToken = resolveContractFromSymbol(tokenSymbol, originChainId);
+  const destinationToken = resolveContractFromSymbol(tokenSymbol, destinationChainId);
+  if (!isDefined(originToken) || !isDefined(destinationToken)) {
+    throw new Error(`Could not resolve token contract for ${tokenSymbol} on ${originChainId} or ${destinationChainId}`);
+  }
+
+  return {
+    // We want to set the deposit ID to 0 so that there
+    // is no chance of a collision with a real deposit
+    depositId: 0,
+    amount: toBN(amountToRelay),
+    depositor: relayerAddress ?? ZERO_ADDRESS,
+    destinationChainId: Number(destinationChainId),
+    originChainId: Number(originChainId),
+    message: messagePayload?.message ?? "0x",
+    originToken,
+    destinationToken,
+    recipient: messagePayload?.recipientAddress ?? ZERO_ADDRESS,
```

Comment:
Same here, zero address will underestimate costs. If we use a default address, we should probably make sure it has no balances, so that token transfers are more costly.

Reply:
Resolved in https://github.com/across-protocol/sdk-v2/pull/427/commits/d94450934f46b17d6fdd0a9ce687378490b4bb64 as the recipient address is now required.

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/427/commits/c08699837a7c78648734ff969f0a74fa275ef890